ci/pipelines/deploy-to-test.yml

```diff
@@ -4090,33 +4090,35 @@ jobs:
           image: selfservice-ecr-registry-test/image.tar
           additional_tags: selfservice-ecr-registry-test/tag
 
-  - name: push-cardid-candidate-to-test-ecr
+  - name: build-and-push-cardid-candidate
     plan:
-      - get: pay-ci
-      - get: cardid-git-release
-        trigger: true
-        params:
-          submodules: none
+      - in_parallel:
+        - get: pay-ci
+        - get: cardid-git-release
+          trigger: true
+          params:
+            submodules: none
       - task: update-submodule
         config:
           container_limits: {}
           platform: linux
           image_resource:
             type: registry-image
             source:
-              repository: govukpay/concourse-runner
+              repository: alpine
           inputs:
             - name: cardid-git-release
           params:
             GH_ACCESS_TOKEN: ((github-access-token))
           outputs:
-            - name: image
+            - name: cardid-git-release-with-submodules
           run:
-            path: bash
+            path: ash
```

Comment:
Is this alpine shell?

Reply:
Yeah

Reply:
I think in alpine sh and ash are actually the same, but ash is slightly different to the sh you get with ubuntu I believe so I like to call out that it's ash not sh really